core/spec/helpers/products_helper_spec.rb

```diff
@@ -217,7 +223,33 @@ module Spree
           allow(Date).to receive(:today) { today }
         end
 
-        it { is_expected.to eq('en/USD/spree/products/all-10-20131211-1234567') }
+        it do
+          is_expected.
+            to eq("en/USD/#{zone.cache_key}/spree/products/all-10-20131211-1234567")
+        end
+      end
+    end
+
+    context '#cache_key_for_product' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.